scipy/stats/_distn_infrastructure.py

```diff
@@ -1429,12 +1519,17 @@ def std(self, *args, **kwds):
         res = sqrt(self.stats(*args, **kwds))
         return res
 
-    def interval(self, alpha, *args, **kwds):
+    def interval(self, confidence=None, *args, **kwds):
         """Confidence interval with equal areas around the median.
 
+        .. deprecated:: 1.8.0
+           Parameter `alpha` will be removed in SciPy 1.10.0, it is replaced by
+           parameter `confidence` because the latter works even when
```

Comment:
```suggestion
           Parameter `alpha` will be removed two releases later; it is replaced
           by parameter `confidence` because the latter works even when
```